test/src/commands/create_test.dart

```diff
@@ -126,7 +126,14 @@ void main() {
               '.tmp',
             ),
           ),
-          vars: {'project_name': 'my_app'},
+          vars: {
+            'project_name': 'my_app',
+            'org_name': [
+              {'value': 'com', 'delimiter': '.'},
+              {'value': 'example', 'delimiter': '.'},
+              {'value': 'verygoodcore', 'delimiter': ''}
```

Comment:
can we simplify this to:

```suggestion
            'org_name': [
              'com',
              'example',
              'verygoodcore',
```